ibm_catalog.json

```diff
@@ -285,6 +285,38 @@
 							"default_value": "__NULL__",
 							"description": "The CRN of an existing KMS instance to use in this solution. If not set, a new KP instance is provisioned.",
 							"required": false
+						},
+						{
+							"key": "resource_groups_scope",
+							"type": "list",
+							"default_value": [],
+							"description": "The resource group to associate with the Security and Compliance Center profile attachments. If not specified, the attachments are scoped to the current account ID. Only one resource group is allowed.",
+							"required": false
+						},
+						{
+							"key": "attachment_schedule",
+							"type": "string",
+							"default_value": "daily",
```

Comment:
We'd better set default to "monthly". This will trigger a first scan right away, but will not rake up daily cost.